### PR TITLE
fail gracefully on Clarinet.toml errors

### PR DESCRIPTION
**Pull request by csgui**
Implementation of an enhancement described by the issue #174 .

The `clarinet check` feature will show an error message if there is something wrong with the `Clarinet.toml` file:

`Error: there is an issue with the Clarinet.toml file`

### CODE REVIEW

File: src/types/project_manifest.rs [hunk 1]
```diff
@@ -72,8 +72,16 @@ impl ProjectManifest {
         project_manifest_file_reader
             .read_to_end(&mut project_manifest_file_buffer)
             .unwrap();
+
         let project_manifest_file: ProjectManifestFile =
-            toml::from_slice(&project_manifest_file_buffer[..]).unwrap();
+            match toml::from_slice(&project_manifest_file_buffer[..]) {
+                Ok(s) => s,
+                Err(_e) => {
+                    println!("Error: there is an issue with the Clarinet.toml file");
```
Comment (lgalabru) [inline src/types/project_manifest.rs:80]:
```suggestion
                    println!("Error: there is an issue with the Clarinet.toml file\n{:?}", _e);
```

**Comment by lgalabru** [comment]
Just had a little suggestion, that could help debugging the error.
Thank you so much for this contribution @csgui!

**Review by obycode — CHANGES_REQUESTED** [review]
Awesome, thanks! Please run `cargo fmt`, then it should be good to go.

File: src/types/project_manifest.rs [hunk 2]
```diff
@@ -72,8 +72,16 @@ impl ProjectManifest {
         project_manifest_file_reader
             .read_to_end(&mut project_manifest_file_buffer)
             .unwrap();
+
         let project_manifest_file: ProjectManifestFile =
-            toml::from_slice(&project_manifest_file_buffer[..]).unwrap();
+            match toml::from_slice(&project_manifest_file_buffer[..]) {
+                Ok(s) => s,
+                Err(_e) => {
+                    println!("{}\n{:?}", red!("Error: there is an issue with the Clarinet.toml file"), _e);
```
Comment (lgalabru) [inline src/types/project_manifest.rs:80]:
```suggestion
                    println!(
                        "{}\n{:?}",
                        red!("Error: there is an issue with the Clarinet.toml file"),
                        _e
                    );
```

**Review by obycode — APPROVED** [review]
👍

**Pull request merged**